Public/Src/Tools/BxlScriptAnalyzer/Args.cs

```diff
@@ -33,6 +33,11 @@ internal sealed partial class Args : CommandLineUtilities
         /// </summary>
         public readonly bool Help;
 
+        /// <summary>
+        /// Serialize using top sort so that loading the graph is faster
+        /// </summary>
+        public readonly bool TopSort;

```

Comment:
This should be property of Analyzer; just like the Engine phases. One use top-sort-capable pip graph if any of the analyzer set the top sort property to true.